### PR TITLE
Feature/mobile navigation header

### PR DESCRIPTION
**Pull request by weberjavi**
This PR creates a mobile navigation menu as proposed by design team. Designs can be checked [here](https://app.zeplin.io/project/598b1dc89689e51e1cd1b5ce/screen/5a5dc4f2556f113ea85894cc) 
Some changes from the original design have been made (after agreement with design team):  
* Nav elements that contain nested routes do not display the arrow icon (are always opened) . 
* An underline is displayed on the actual section to have a navigation reference when the menu is open.  
* Scroll functionality in the home page (that enables changing in opacity to the mobile navbar) is not developed in this stage.  
  
navbarmobile

### CODE REVIEW

File: app/javascript/app/components/navbar-mobile/nav-with-child-menu/nav-with-child-menu.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { connect } from 'react-redux';
+
+import { actions } from 'components/hamburger';
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply app/javascript/app/components/navbar-mobile/nav-with-child-menu/nav-with-child-menu.js]: :-)

File: app/javascript/app/components/hamburger/hamburger-component.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import close from 'assets/icons/sidebar-close.svg';
+import hamburger from 'assets/icons/hamburger.svg';
+
+import styles from './hamburger-styles.scss';
+
+const Hamburger = ({ isOpen, openMenu, closeMenu, text, className }) => (
+  <div className={styles.container}>
+    <p className={cx(styles.text, className)}>{text}</p>
+    {isOpen && (
+      <div
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply app/javascript/app/components/hamburger/hamburger-component.jsx]: I agree

**Comment by Bluesmile82** [comment]
Please take a look at the anchor fixed bar when we scroll down. Not sure if it should go down or just disappear in mobile. We can talk with design about it.
image
image

**Comment by j8seangel** [comment]
There are still some differences with the designs, for example, the nav with child menu still needs styles to match with design, uppercase text, letter-spacing...

After that feel free to merge!

**Comment by weberjavi** [comment]
Addressed design-implementation inconsistencies (letter spacing and casing).  
Also fixed a couple of bugs detected by @Bluesmile82 👍   (sticky nav anchors positioning and share button reverse behaviour)

**Pull request merged**